### PR TITLE
Testing part2

### PR DESCRIPTION
**Pull request by JasemDuncan**
In this part, we created a new test function for Editing a Task, updated status for a completed task, and cleared all completed tasks. Also, we mock the localStorage to implement the testing.

### CODE REVIEW

File: src/test-2.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+const arrObjTask = [
```
Comment (SajjadAhmad14) [inline src/test-2.js:1]: Please add `npm install`, `npm start`, and `npm test` commands in README for a non-technical user.
Please add all unnecessary files and folders like `vscode` to `gitignore`

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Anibal and Jasem, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
All tests are passing. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/test-2.js (hunk 1 above)
Comment (JasemDuncan) [reply src/test-2.js:1]: Thanks for the sugestion. Implementing now.

**Review by EricMbouwe — APPROVED** [review]

**Review by EricMbouwe — APPROVED** [review]
### Hi @JasemDuncan, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**